lib/api/src/rc_get_resource_info_for_operation.cpp

```diff
@@ -0,0 +1,18 @@
+#include "irods/get_resource_info_for_operation.h"
+
+#include "irods/procApiRequest.h"
+#include "irods/apiNumber.h"
+
+auto rc_get_resource_info_for_operation(RcComm* _comm, const DataObjInp* _dataObjInp, char** _out_info) -> int
```

Comment:
Declaration uses `_conn` and this uses `_comm`. I'm voting `_comm` in both places but consistency above all